src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -100,6 +100,7 @@ public function load(array $configs, ContainerBuilder $container)
                     'macvim' => 'mvim://open?url=file://%%f&line=%%l',
                     'emacs' => 'emacs://open?url=file://%%f&line=%%l',
                     'sublime' => 'subl://open?url=file://%%f&line=%%l',
+                    'symfony' => '/_profiler/open?file=%%f&line=%%l#line%%l#"%kernel.root_dir%/=/',
```

Comment:
`'/_profiler/open?file=%%f&line=%%l#line%%l#'.json_encode(dirname($container->getParameter('kernel.root_dir')).DIRECTORY_SEPARATOR).':""',` so that is works on Windows also